Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -394,36 +472,69 @@ def formula_name(formula, full_name: false)
+    sig { params(resource: Resource, full_name: T::Boolean).returns(String) }
+    def resource_name(resource, full_name: false)
+      full_name ? resource.full_name : resource.name
+    end
+
     sig {
       params(
-        formula_or_cask: T.any(Formula, Cask::Cask),
-        status_str:      String,
-        messages:        T.nilable(T::Array[String]),
-        full_name:       T::Boolean,
-        verbose:         T::Boolean,
+        package_or_resource: T.any(Formula, Cask::Cask, Resource),
+        status_str:          String,
+        messages:            T.nilable(T::Array[String]),
+        full_name:           T::Boolean,
+        verbose:             T::Boolean,
       ).returns(Hash)
     }
-    def status_hash(formula_or_cask, status_str, messages = nil, full_name: false, verbose: false)
-      formula = formula_or_cask if formula_or_cask.is_a?(Formula)
-      cask = formula_or_cask if formula_or_cask.is_a?(Cask::Cask)
+    def status_hash(package_or_resource, status_str, messages = nil, full_name: false, verbose: false)
+      formula = package_or_resource if package_or_resource.is_a?(Formula)
+      cask = package_or_resource if package_or_resource.is_a?(Cask::Cask)
+      resource = package_or_resource if package_or_resource.is_a?(Resource)
 
       status_hash = {}
       if formula
         status_hash[:formula] = formula_name(formula, full_name: full_name)
       elsif cask
-        status_hash[:cask] = cask_name(formula_or_cask, full_name: full_name)
+        status_hash[:cask] = cask_name(package_or_resource, full_name: full_name)
+      elsif resource
+        status_hash[:resource] = resource_name(package_or_resource, full_name: full_name)
       end
       status_hash[:status] = status_str
       status_hash[:messages] = messages if messages.is_a?(Array)
 
       status_hash[:meta] = {
-        livecheckable: formula_or_cask.livecheckable?,
+        livecheckable: package_or_resource.livecheckable?,
       }
       status_hash[:meta][:head_only] = true if formula&.head_only?
 
       status_hash
     end
 
+    # Formats and prints the livecheck result for a resource (for a given Formula or Cask).
+    sig { params(resources_info: Array(Hash), verbose: T::Boolean, ambiguous_cask: T::Boolean).void }
+    def print_latest_resource_version(resources_info, verbose:, ambiguous_cask: false)
+      resources_info.each_with_index do |info, _i|
+        resource_s = "#{Tty.blue}#{info[:resource]}#{Tty.reset}"
+        resource_s += " (livecheckable)" if info[:livecheckable] && verbose
+
+        current_s = if info[:version][:newer_than_upstream]
+          "#{Tty.red}#{info[:version][:current]}#{Tty.reset}"
+        else
+          info[:version][:current]
+        end
+
+        latest_s = if info[:version][:outdated]
+          "#{Tty.green}#{info[:version][:latest]}#{Tty.reset}"
+        else
+          info[:version][:latest]
+        end
+
+        puts "-- #{resource_s}: #{current_s} ==> #{latest_s}"
```

Comment:
```suggestion
        puts "  #{resource_s}: #{current_s} ==> #{latest_s}"
```

I tend to agree with Nanda's suggestion on Slack of simply indenting resources by two spaces, unless someone has a more compelling idea.

Reply:
Done ✅